source/api/presentation/3.0/index.md

```diff
@@ -518,12 +507,13 @@ The value _MUST_ be a string, taken from the table below or a full URI.
 ```
 
 ##### behavior
-A set of user experience features for the client to use when presenting this resource to the user. This specification defines the values specified in the table below. Others may be defined externally, and would be given as a URI.
 
-The value _MUST_ be an array of strings, taken from the table below or a URI.
+A set of user experience features that the publisher of the content would prefer the client to use when presenting this resource. This specification defines the values specified in the table below. Others may be defined externally, and _SHOULD_ be [registered as an extension][registry-home].
```

Comment:
Should we make the distinction between local (not to be registered) and community endorsed (to be registered) extensions here?